examples/Console/AccessingPolymorphicType.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Sinch;
+using Sinch.Verification.Start.Request;
+using Sinch.Verification.Start.Response;
+
+namespace Examples
+{
+    public class AccessingPolymorphicType
+    {
+        public static void Example()
+        {
+            var sinchClient = new SinchClient("KEY_ID", "KEY_SECRET", "PROJECT_ID");
+            var verificationStart = sinchClient.Verification("APP_KEY", "APP_SECRET").Verification
```

Comment:
You may want to rename this variable `verificationStartResponse` to be consistent with the other variables below or `verificationStartResult` to be consistent with the function call.